src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs

```diff
@@ -376,29 +374,25 @@ private void UpdateInputScope(InputScope inputScope)
 			}
 		}
 
-		partial void OnIsSpellCheckEnabledChangedPartial(DependencyPropertyChangedEventArgs e)
+		partial void OnIsSpellCheckEnabledChangedPartial(bool newValue)
 		{
 			if (_textBoxView != null)
 			{
 				UpdateInputScope(InputScope);
 			}
 		}
 
-		partial void OnAcceptsReturnChangedPartial(DependencyPropertyChangedEventArgs e)
+		partial void OnAcceptsReturnChangedPartial(bool newValue)
 		{
-			var acceptsReturn = (bool)e.NewValue;
-			_textBoxView?.SetHorizontallyScrolling(!acceptsReturn);
+			_textBoxView?.SetHorizontallyScrolling(!newValue);
 			_textBoxView?.UpdateSingleLineMode();
 
 			UpdateInputScope(InputScope);
 		}
 
-		partial void OnTextWrappingChangedPartial(DependencyPropertyChangedEventArgs e)
+		partial void OnTextWrappingChangedPartial(TextWrapping newValue)
```

Comment:
the `newValue` argument is not used and could be removed

Reply:
Sure, can remove them. But let's first see if we even want to take this PR or not.